### DISCUSSION TITLE
Is there a way to use the EPUB media overlays instead of TTS for the `Speak from Here` feature?

### DISCUSSION THREAD

**Discussion by macr** [johnfactotum/foliate#1429] [Q&A]
If that's not an option maybe a way to start the narration from a section on the page?

Example ebook:
https://www.readbeyond.it/samples/ad1db71e.epub

Here's how it looks/reads on readium:
https://www.readbeyond.it/readium/?epub=epub_content/ad1db71e

**Comment by johnfactotum** [comment]
No. It's a missing feature.
